.github/workflows/bundleTest-thor.md

```diff
@@ -0,0 +1,353 @@
+```yaml
+env:
+  ML_SUPPRESS_WARNING_FILES: "RegressionTestModified.ecl ClassificationTestModified.ecl"
+  ML_EXCLUDE_FILES: "--ef ClassicTestModified.ecl,SVCTest.ecl,ClassificationTestModified.ecl"
+  BUNDLES_TO_TEST: "ML_Core PBblas GLM  GNN DBSCAN LearningTrees TextVectors KMeans SupportVectorMachines LinearRegression LogisticRegression"       
+  uploadArtifact: false 
+```
+- `ML_SUPPRESS_WARNING_FILES:` Specifies the files requiring a warning suppression parameter during runtime.
+- `ML_EXCLUDE_FILES:` The files specified here are excluded during the run.
+- `BUNDLES_TO_TEST:` Lists the bundles to test.
+- `uploadArtifact:` Determines whether the logs Artifact should be uploaded or not.
+## Steps Involved:
+The steps in the workflow run on the specified operating system, with Ubuntu-22.04 as the default.
+- **Download Package**  
+This step enables us to download the ready-to-install HPCC Platform's artifact built on the latest commit.
+  ```yaml
+      - name: Download Package
+          uses: actions/download-artifact@v3
+          with:
+            name: ${{ inputs.asset-name }}
+            path: ${{ inputs.asset-name }}
+  ```
+- **Install Dependencies**  
+Install the dependencies that are necessary for the platform to install and start sucessfully.
+  ```yaml
+  - name: Install Dependencies
+      shell: "bash"
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+          git \
+          wget \
+          net-tools \
+          tzdata \
+          unzip \
+          xvfb \
+          libxi6 \
+          default-jdk \
+          gdb \
+          ${{ inputs.dependencies }}
+  ```
+- **Install Package**  
+The HPCC Platform is installed from the downloaded artifact.
+  ```yaml
+  - name: Install Package
+    shell: "bash"
+    run: |
+      sudo apt-get install -y -f ./${{ inputs.asset-name }}/*.deb
+      sudo chown -R $USER:$USER /opt/HPCCSystems
+      sudo xmlstarlet ed -L -u 'Environment/Software/ThorCluster/@slavesPerNode' -v 2 -u 'Environment/Software/ThorCluster/@channelsPerSlave' -v 1 /etc/HPCCSystems/environment.xml
+  ```
+- **Install ML Dependencies**  
+Install the necessary Machine learning library dependecies
+  ```yaml
+  - name: Install ML Dependencies
+      shell: "bash"
+      run: |  
+        sudo apt install libsvm-dev libsvm-tools 
+        sudo pip install tensorflow numpy keras   
+  ```
+- **Start HPCC-Platform**  
+Setup the core generation and start the HPCC Platform
+  ```yaml
+  - name: Start HPCC-Platform
+          shell: "bash"
+          run: |
+            export LANG="en_US.UTF-8"
+            ulimit -c 100
+            echo 'core_%e.%p' | sudo tee /proc/sys/kernel/core_pattern
+            sudo update-locale
+            sudo /etc/init.d/hpcc-init start
+  ```
+- **Core generation test**  
+This step is used to check if the core generation is working as expected. This is an optional step. This step runs only when it is specified in the inputs via workflow call.
```

Comment:
A couple of words about how it is working and where the core file will be generated would be nice.